version.sbt

```diff
@@ -1 +1 @@
-version in ThisBuild := "1.0.2-SNAPSHOT"
+version in ThisBuild := "1.0.3-SNAPSHOT"
```

Comment:
Hey @hughsimpson, maybe you ran a local release or something? this file should be showing 1.0.2-SNAPSHOT :)